OpenTabletDriver.Web/Views/Wiki/FAQ/Linux.cshtml

```diff
@@ -2,67 +2,122 @@
-                    sudo rmmod wacom
-                </codeblock>
-            </li>
-        </ol>
+    </p>
+    <h4>It is a supported tablet, but still doesn't detect after a reboot?</h4>
+    <p>
+        If you are running OpenTabletDriver.UX, navigate from the top to the Console tab, if you are running the daemon standalone, look for a log that resembles the following:<br/>
+
+    <h5 class="wiki-nav-item pb-2" id="hid_uclogic">hid_uclogic kernel module interefence</h5>
+        <codeblock language="bash">
+        Another tablet driver found: UC Logic
+        ArgumentOutOfRangeException Value range is [0, 15]. (Parameter 'value')
+        </codeblock>
+        These errors may happen if the <code>hid_uclogic</code> kernel module is loaded at the same time as OpenTabletdriver, you can remove
+        and blacklist this module to stop it from loading in the future by running the following commands:
+        <codeblock class="mt-2", language="bash">
+            echo "blacklist hid_uclogic" | sudo tee -a /etc/modprobe.d/blacklist.conf
+            sudo rmmod hid_uclogic
+        </codeblock>
+
+    <h5 class="wiki-nav-item pb-2" id="udev">Missing udev rules</h5>
+        <codeblock language="bash">
+            Not permitted to open HID class device at /dev/hidrawX
+        </codeblock>
+        If you have followed the previous instructions, usually this is the only reason OTD will fail to detect your tablet when its supported.
+        The issue is resolved my making sure that you have proper udev rules set up. If you installed OpenTabletDriver via your package manager,
+        you may just need to reload and trigger the rules. You can make sure of this by following the instructions below.
+
+        <codeblock class="mt-2" language="bash">
+            sudo udevadm control --reload-rules
+            sudo udevadm trigger # Or replug your tablet.
+        </codeblock>
+
+        If it still doesn't work after this point, but you installed OpenTabletDriver by a package manager,
+        please reboot, some systems may require this.<br/>
+
+        If you installed OTD without using a package manager (we <b>highly</b> recommend you use a packaged build if available),
+        you will not have proper udev rules set up and must build your own. This process will require you to have the dotnet 7 SDK:
+        <codeblock class="mt-2" language="bash">
+            # Clone the repository, change current directory to the repository
+            git clone https://github.com/OpenTabletDriver/OpenTabletDriver.git
+            cd ./OpenTabletDriver
+            # Generate rules, moves them to the udev rules directory
+            ./generate-rules.sh
+            sudo mv ./bin/99-opentabletdriver.rules /etc/udev/rules.d/99-opentabletdriver.rules
+            # Reload udev rules
+            sudo udevadm control --reload-rules
+            # Clean up leftovers
+            cd ..
+            rm -rf OpenTabletDriver
+        </codeblock>
```

Comment:
should probably add to come to our Discord if it still doesn't work since there could actually be a packaging issue (it has happened before)

Reply:
nvm this is in the bottom (can't mark as solved for some reason)